README.md

```diff
@@ -46,11 +46,12 @@ The operations we'll perform:
 
 ### Registering with the API
 
-#### Demo
+#### Demonstration: Follow Along
```

Comment:
How about `Demonstration: Write Code Methodically` or `Demonstration: Use an Echo Server to Help Write Code`?